### DISCUSSION TITLE
Mamba installation

### DISCUSSION THREAD

**Discussion by mtimana** [giswqs/geog-312#31] [Q&A] [answered]
Hello Dr. Wu. I am following your course on GIS programming. I was following the instructions to install Mamba but I keep getting this error message:
Preparing transaction: done
Verifying transaction: failed

EnvironmentNotWritableError: The current user does not have write permissions to the target environment.
  environment location: C:\ProgramData\miniconda3

Any suggestion as to how to solve it and keep going? 
Thank you very much in advance.

Martin Timana
Lima, Peru

**Comment by giswqs** [comment, marked as answer]
```bash
conda create -n geo python=3.11
conda activate geo
conda install mamba -c conda-forge
mamba install -c conda-forge geemap leafmap
```

**Reply by mtimana** [reply]
it worked! Thanks a lot!
